### PR TITLE
fix(1500): [1] add unzipArtifacts method

### PR DESCRIPTION
**Pull request by s-yoshika**
## Context
To implement SD_ZIP_ARTIFACTS feature by [new design](https://github.com/screwdriver-cd/screwdriver/blob/0962d20d251c0b7c03d40b7e4b10bb1ed009831d/design/sd-zip-artifacts.md)

Please refer the following for the progress of this feature.
https://github.com/screwdriver-cd/screwdriver/issues/1500#issuecomment-998382248

## Objective
This PR add a new method for executor to unzip artifacts.
`SD_ZIP_ARTIFACTS` feature requires `queue-service` component according to the new design, so this new method is supposed to be supported only by `executor-queue` plugin.

## References
[design doc](https://github.com/screwdriver-cd/screwdriver/blob/0962d20d251c0b7c03d40b7e4b10bb1ed009831d/design/sd-zip-artifacts.md#sd-api-api)

[issue](https://github.com/screwdriver-cd/screwdriver/issues/1500)

## License

I confirm that this contribution is made under the terms of the license found in the root directory of this repository's source tree and that I have the authority necessary to make this contribution on behalf of its copyright owner.

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -241,6 +241,20 @@ class Executor {
 
         return parsedAnnotations;
     }
+
+    /**
+     * Unzip the ZIP of artifacts
+     * @method unzipArtifacts
+     * @param  {Object} config       Configuration
```
Comment (sakka2) [inline index.js:248]: need param `config.buildId` like executor-queue.
https://github.com/sonic-screwdriver-cd/executor-queue/blob/30e75c892f5520c4c100d33029ea49572da239a6/index.js#L140

Comment (s-yoshika) [reply index.js:248]: Thanks, I didn't understand the param, which isn't refered in this function, also should be written. I'll fix.

**Review by sakka2 — APPROVED** [review]
LGTM

**Review by tk3fftk — APPROVED** [review]

**Review by esuc4se — APPROVED** [review]
LGTM

**Review by tkyi — APPROVED** [review]

**Pull request merged**